man/man/man5/cheatsheet.5

```diff
@@ -0,0 +1,692 @@
+.\" te
+.\"  Copyright (c) 2013, Joyent, Inc. All Rights Reserved
+.\" The contents of this file are subject to the terms of the Common Development and Distribution License (the "License"). You may not use this file except in compliance with the License.
+.\" You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE or http://www.opensolaris.org/os/licensing. See the License for the specific language governing permissions and limitations under the License.
+.\" When distributing Covered Code, include this CDDL HEADER in each file and include the License file at usr/src/OPENSOLARIS.LICENSE. If applicable, add the following below this CDDL HEADER, with the fields enclosed by brackets "[]" replaced with your own identifying information: Portions Copyright [yyyy] [name of copyright owner]
+.TH CheatSheet 5 "27 May 2013" "SmartOS" "Linux to SmartOS Cheat Sheet"
+.SH NAME
+.B Linux to SmartOS Zone CheatSheet
+.P
+.SH SYNOPSIS
+.B CheatSheet 
+For Mapping \fBLinux\fR 
+.I man
+commands to \fBSmartOS\fR 
+.I man 
+commands
+
+
+.SH AVAILABILITY
+Applies Within A SmartOS Zone 
+.B = SmartMachine Virtual Instance
+.P
+This CheatSheet \fBdoes not apply to the SmartOS Global Zone\fR.
+.nf
+
+Linux Specific Key
+(rh)    Red Hat, CentOS
+(deb)   Debian
+(ub)    Ubuntu
+(fed)   Fedora
+(SUSE)  SUSE
+
+.fi
+
+Linux man section (1) are executable or shell commands, whereas section (8) specifies 
+system administration commands that are run by root. SmartOS man pages are (1) for general
+commands or (1M) for commands and daemons that are primarily used for system maintenance 
+and administrative purposes.
+
+
+.SH DESCRIPTION
+SmartOS is an open source illumOS operating system distribtuion provided by Joyent.
+It is optimized for enterprise grade cloud-based hypervisor services. illumOS based 
```

Comment:
It's specifically a lower case i because of fonts that cause the capital i character to look like a lowercase l character.